### ISSUE TITLE
TypeError: e.scrollIntoViewIfNeeded is not a function

### ISSUE THREAD

**Issue by jurgenhaas** [bpmn-io/element-template-chooser#10]
__Describe the Bug__

Just updated to the latest version 0.0.3 - great work everyone and big thank you!

The only issue: when clicking on "+ Select" for a template, the template chooser opens but the browser console outputs this exception:

```
Uncaught (in promise) TypeError: e.scrollIntoViewIfNeeded is not a function
    AR https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    SR https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    __h https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    __c https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    __c https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    JA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    A https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    $A https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    FA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    setState https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    __ https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    AR https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    SR https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    wR https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    setTimeout handler*n https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    diffed https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    diffed https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    XA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    HA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    __e https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    XA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    HA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    __e https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    XA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    HA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    XA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    HA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    XA https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    tR https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    _refresh https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:74
    open https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:82
    open https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:62
    TR https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    r https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    _invokeListener https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    _invokeListeners https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    fire https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    onClick https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Wg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Gg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Jg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Jg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Jg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Jg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Jg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Jg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Jg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Yg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
    Fg https://danse.dev.jurgenhaas.de/modules/contrib/bpmn_io/webpack/bpmn-modeller.js?v=8.7.1:2
```

Selecting a template with the mouse works, but keyboard interaction doesn't. I can't scroll with the keyboard and search doesn't work either. Both is certainly a follow-up error off the exception.

__Environment__

Library versions:

- diagram-js@8.2.1
- camunda-bpmn-moddle@6.1.2
- min-dash@3.8.1
- jquery@3.6.0
- bpmn-js@9.0.3
- @bpmn-io/element-template-chooser@0.0.3
- @bpmn-io/properties-panel@0.12.0
- bpmn-js-properties-panel@1.0.0-alpha.7

**Comment by rpkoller** [comment]
probably related with this issue https://github.com/bpmn-io/element-template-chooser/issues/6 and the commit fixing it: https://github.com/bpmn-io/element-template-chooser/commit/44b16fbac9014170f4b92c25be55e0d2101407fa . `src/element-template-chooser/ElementTemplateChooser.js` gets added that function shown in the typeerror in line 141

**Comment by nikku** [comment]
Please provide all details, including your browser environment.

**Comment by jurgenhaas** [comment]
I'm on Ubuntu 20.04 with Firefox 98.0.2

**Comment by nikku** [comment]
Released with `v0.0.4`.
